openmc/capi/filter.py

```diff
@@ -241,6 +265,26 @@ class PolarFilter(Filter):
     filter_type = 'polar'
 
 
+class SphericalHarmonicsFilter(Filter):
+    filter_type = 'sphericalharmonics'
+
+    @property
+    def order(self):
```

Comment:
Would also be good to add a property setter for `order` on all the expansion filter classes here